OpenRA.Mods.Common/Widgets/Logic/Ingame/Hotkeys/CycleBasesHotkeyLogic.cs

```diff
@@ -41,6 +41,8 @@ protected override bool OnHotkeyActivated(KeyInput e)
 
 			var bases = world.ActorsHavingTrait<BaseBuilding>()
 				.Where(a => a.Owner == player)
+				.OrderByDescending(a => a.IsPrimaryBuilding())
+				.ThenByDescending(a => a.ActorID)
```

Comment:
This changes the behavior we have now (cycling from oldest to newest conyard). I think it makes more sense to cycle from oldest to newest.

Reply:
does it? ActorID should reflect the age of the actor

Reply:
Fixed.